src/sentry/tasks/integrations/github/constants.py

```diff
@@ -0,0 +1,58 @@
+ISSUE_LOCKED_ERROR_MESSAGE = "Unable to create comment because issue is locked."
+
+RATE_LIMITED_MESSAGE = "API rate limit exceeded"
+
+# ---
+
+# MERGED PR COMMENTS
+
+MERGED_PR_METRICS_BASE = "github_pr_comment.{key}"
+
+MERGED_PR_COMMENT_BODY_TEMPLATE = """## Suspect Issues
```

Comment:
I usually find these easier to read if written as:

```python
x = """\
first line here
...
...
"""
```

the escaped newline at the beginning makes it so the whole string is readably pushed to the edge

Reply:
TIL